modules/apm/main.tf

```diff
@@ -1,15 +1,19 @@
-resource "newrelic_alert_policy" "main_policy" {
-  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] ${var.policy_name}"
+#
+# Business hours alert policy
+#
+
+resource "newrelic_alert_policy" "bh_policy" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] ${var.policy_name} (BH)"
 }
 
-resource "newrelic_nrql_alert_condition" "app_apdex_score_alert" {
-  count       = var.app_apdex_score_enabled == true ? 1 : 0
+resource "newrelic_nrql_alert_condition" "app_apdex_score_bh_alert" {
+  count       = var.app_apdex_score_enabled && ! var.app_apdex_score_nbh_alerts_enabled ? 1 : 0
```

Comment:
should not it be `app_apdex_score_bh_alerts_enabled` here ?

Reply:
Removed in new version